CMakeLists.txt

```diff
@@ -364,11 +364,41 @@ if(WIN32)
     list(APPEND QUIC_COMMON_DEFINES QUIC_DISABLE_SHARED_PORT_TESTS)
 
     if (QUIC_UWP_BUILD)
-        list(APPEND QUIC_COMMON_DEFINES WINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP QUIC_UWP_BUILD)
+        list(APPEND QUIC_COMMON_DEFINES WINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP QUIC_RESTRICTED_BUILD)
+        set(CMAKE_CXX_STANDARD_LIBRARIES "")
+        set(CMAKE_CXX_STANDARD_LIBRARIES_INIT "")
+        set(CMAKE_C_STANDARD_LIBRARIES "")
+        set(CMAKE_C_STANDARD_LIBRARIES_INIT "")
     endif()
 
     if (QUIC_GAMECORE_BUILD)
-        list(APPEND QUIC_COMMON_DEFINES WINAPI_FAMILY=WINAPI_FAMILY_GAMES QUIC_GAMECORE_BUILD)
+        list(APPEND QUIC_COMMON_DEFINES WINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP QUIC_RESTRICTED_BUILD)
+        set(CMAKE_CXX_STANDARD_LIBRARIES "")
+        set(CMAKE_CXX_STANDARD_LIBRARIES_INIT "")
+        set(CMAKE_C_STANDARD_LIBRARIES "")
+        set(CMAKE_C_STANDARD_LIBRARIES_INIT "")
+
+        set(UnsupportedLibs advapi32.lib comctl32.lib comsupp.lib dbghelp.lib gdi32.lib gdiplus.lib guardcfw.lib kernel32.lib mmc.lib msimg32.lib msvcole.lib msvcoled.lib mswsock.lib ntstrsafe.lib ole2.lib ole2autd.lib ole2auto.lib ole2d.lib ole2ui.lib ole2uid.lib ole32.lib oleacc.lib oleaut32.lib oledlg.lib oledlgd.lib oldnames.lib runtimeobject.lib shell32.lib shlwapi.lib strsafe.lib urlmon.lib user32.lib userenv.lib wlmole.lib wlmoled.lib onecore.lib)
```

Comment:
Where did you get this list? Why is `onecore.lib` on it?

Reply:
https://github.com/microsoft/Xbox-GDK-Samples/blob/main/Samples/Tools/CMakeExample/XboxOne/CMakeGDKXboxOne.cmake#L199